Library/Formula/curlish.rb

```diff
@@ -0,0 +1,15 @@
+class Curlish < Formula
+  homepage "https://pythonhosted.org/curlish/"
+  url "https://github.com/fireteam/curlish/archive/1.22.tar.gz"
+  sha1 "45e9a7d92b5f70adf257cda3f9f0f205eef0245b"
+
+  depends_on :python if MacOS.version <= :snow_leopard
```

Comment:
@mikemcquaid So this line is only for formulae which need Python 2.7? I thought it was needed for all Python-based formulae.

I just tested, and it doesn’t work with Python 2.6 because it uses `argparse`, which was added in 2.7 (there’s a backport on Pypi but they don’t use it).